.gitpod.yml

```diff
@@ -0,0 +1,7 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
This file isn't related to this change, let's leave it out of the PR.

Reply:
great idea, thank you